flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/casting/CastRulesTest.java

```diff
@@ -656,13 +656,6 @@
                         .fromCaseLegacy(CHAR(6), fromString("Apache"), fromString("Apache"))
                         .fromCase(VARCHAR(5), fromString("Flink"), fromString("Flink "))
                         .fromCaseLegacy(VARCHAR(5), fromString("Flink"), fromString("Flink"))
-                        // We assume that the input length is respected, therefore, no trimming is
-                        // applied
```

Comment:
Update the hotfix commit if those tests can be removed? They are invalid anyways so +1.

Reply:
Yep I will do.